runtime/src/main/java/org/corfudb/runtime/Restore.java

```diff
@@ -205,19 +223,17 @@ private void restoreTable(Path filePath, UUID streamId) throws IOException {
     /**
      * Check if the table has requires_backup_tag. Return true if it's RegistryTable.
      */
-    private boolean isTableTagged(String tableBackup) {
-        // tableBackup name format: uuid.namespace$tableName
-        UUID uuid = UUID.fromString(tableBackup.substring(0, tableBackup.indexOf(".")));
+    private boolean isTableTagged(TableBackupFileName tableBackupFileName) {
```

Comment:
nit: For clarity, add a comment to explain this is calculated based off of the restored registryTable

Reply:
Done.